apollo-router/src/plugins/snapshots/apollo_router__plugins__expose_query_plan__tests__it_expose_query_plan_without_executing.snap

```diff
@@ -0,0 +1,173 @@
+---
+source: apollo-router/src/plugins/expose_query_plan.rs
+expression: "serde_json::to_value(response).unwrap()"
+---
+{
+  "extensions": {
+    "apolloQueryPlan": {
+      "object": {
+        "kind": "QueryPlan",
```

Comment:
Is there an entirely different, more precise, assertion we could do here or different way to test this that doesn't encode the actual query plan in this?  Right now, this would require updating this snapshot every time the query plan changes, but also every time the `schemaAwareHash` changes?

Reply:
yep! removing this file